modules/distr-tracing-tempo-config-receiver-tls-for-tempomonolithic.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * observability/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-configuring.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="distr-tracing-tempo-config-receiver-tls-for-tempomonolithic_{context}"]
+= Configuring the receiver TLS for a TempoMonolithic instance
+
+* You can provide a TLS certificate in a secret and configure it in the `TempoMonolithic` custom resource.
++
+[NOTE]
+====
+This feature is not supported with the enabled Tempo Gateway.
+====
++
+.TLS for receivers and using a user-provided certificate in a secret
+[source,yaml]
+----
+ apiVersion: tempo.grafana.com/v1alpha1
+ kind:  TempoMonolithic
+# ...
+    spec:
+# ...
+      ingestion:
```

Comment:
GRPC and HTTP both support TLS configs:
```
  ingestion:
    otlp:
      grpc:
        tls:
          enabled: true
      http:
        tls:
          enabled: true
```
https://github.com/grafana/tempo-operator/blob/main/tests/e2e-openshift/tls-monolithic-singletenant/01-install-tempo.yaml#L11